Packs/Tessian/Integrations/Tessian/Tessian.py

```diff
@@ -0,0 +1,167 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+"""Base Integration for Cortex XSOAR (aka Demisto)
+
+This is an empty Integration with some basic structure according
+to the code conventions.
+
+MAKE SURE YOU REVIEW/REPLACE ALL THE COMMENTS MARKED AS "TODO"
+
+Developer Documentation: https://xsoar.pan.dev/docs/welcome
+Code Conventions: https://xsoar.pan.dev/docs/integrations/code-conventions
+Linting: https://xsoar.pan.dev/docs/integrations/linting
+
+This is an empty structure file. Check an example at;
+https://github.com/demisto/content/blob/master/Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.py
+
+"""
+
+from CommonServerUserPython import *  # noqa
+
+import urllib3
+from typing import Any
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    """Client class to interact with the service API
+
+    This Client implements API calls, and does not contain any XSOAR logic.
+    Should only do requests and return data.
+    It inherits from BaseClient defined in CommonServer Python.
+    Most calls use _http_request() that handles proxy, SSL verification, etc.
+    For this  implementation, no special attributes defined
+    """
+
+    # TODO RETURNS LIST
+    def get_events(self, limit: str | None, after_checkpoint: str | None, created_after: str | None) -> dict[str, Any]:
+        return self._http_request(
+            method='GET',
+            url_suffix='/api/v1/events',
+            data=(
+                ('limit', limit),
+                ('after_checkpoint', after_checkpoint),
+                ('created_after', created_after)
+            ),
+            resp_type='json',
+            ok_codes=(200,)
+        )
+
+
+''' HELPER FUNCTIONS '''
+
+''' COMMAND FUNCTIONS '''
+
+
+def get_events_command(client: Client, args: dict[str, Any]) -> CommandResults:
+    pass
+    limit = args.get('limit', None)
+    after_checkpoint = args.get('after_checkpoint', None)
+    created_after = args.get('created_after', None)
+
+    result = client.get_events(limit, after_checkpoint, created_after)
+    output_results = {
+        "checkpoint": result['checkpoint'],
+        "additional_results": result['additional_results'],
+    }
+    raw_results = result['results']
+
+    return CommandResults(
```

Comment:
I couldn't see where I could specify a list of sorts in the spec, and I don't want to give a 1:1 mapping of our API spec to the return here otherwise we have to update the integration everytime we add a new field to our APIs. So I've opted to put the response in the `raw_response` field